_data/public-speaking.yml

```diff
@@ -1,3 +1,33 @@
+- name: ""
+  by: ""
+  event: ""
+  subTitle: ""
+  date: 
+  summary: >
+    
+  content: ""
+  level: 
+  duration: ""
+  image: ""
+  linkedIn: ""
+
+- name: "Redux Reborn: Embracing Modern State Management with Redux Toolkit (RTK) v5"
+  by: "Wouter"
+  event: "devs.gent"
+  date: 2025-01-29
+  summary: >
+    Wouter Van Schandevijl from Itenium led a deep dive into 
+    ReduxJS/Toolkit—the modern way to handle state management—at 
+    Aikido, 9000 Gent. Starting with a classic "TODO App," he 
+    guided attendees through the Mythical Zoos and Enchanted 
+    Gardens of RTK Query and createAsyncThunk. A great session 
+    filled with hands-on learning and fun!
```

Comment:
A great session demonstrating the power of the toolkit, filled with practical tips on getting started and migrating from legacy Redux.